applications/ray/kuberay-tpu-webhook/deployments/validate-raycluster-webhook-cfg.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validate-raycluster-webhook-cfg
```

Comment:
Instead of introducing a new webhook,  you can specify multiple webhooks in `validating-kuberay-tpu-webhook-cfg` 
```
apiVersion: admissionregistration.k8s.io/v1
kind: ValidatingWebhookConfiguration
metadata:
  name: validating-kuberay-tpu-webhook-cfg
  annotations:
    cert-manager.io/inject-ca-from: default/kuberay-tpu-webhook-certs
webhooks:
  - name: pods-kuberay-tpu-webhook.default.svc
    admissionReviewVersions: [v1]
    sideEffects: NoneOnDryRun
    clientConfig:
      service:
        name: kuberay-tpu-webhook
        namespace: ray-system
        path: /validate
    rules:
      - operations: ["DELETE"]
        apiGroups: [""]
        apiVersions: ["v1"]
        resources: ["pods"]
    objectSelector:
      matchLabels:
        app.kubernetes.io/name: kuberay
  - name: raycluster-kuberay-tpu-webhook.default.svc
    admissionReviewVersions: [v1]
    sideEffects: NoneOnDryRun
    clientConfig:
      service:
        name: kuberay-tpu-webhook
        namespace: ray-system
        path: /validate
    rules:
      - operations: ["CREATE"]
        apiGroups: ["ray.io"]
        apiVersions: ["*"]
        resources: ["rayclusters"]
```